il-operator/controller/services/operations/argocd/operation.go

```diff
@@ -286,3 +291,56 @@ func RegisterNewCluster(ctx context.Context, k8sClient awseks.API, argocdClient
 
 	return info, nil
 }
+
+func RegisterInCluster(ctx context.Context,
+	argocdClient argocdapi.API,
+	cluster string,
+	namespaces []string,
+	log *logrus.Entry) error {
+
+	tokenResponse, err := argocdClient.GetAuthToken()
+	if err != nil {
+		return errors.Wrap(err, "error getting auth token")
+	}
+	bearer := toBearerToken(tokenResponse.Token)
+
+	log.Infof("Checking does k8s cluster %s exist", cluster)
+	clusters, err := argocdClient.ListClusters(&cluster, bearer)
+	if err != nil {
+		return errors.Wrapf(err, "error listing clusters")
+	}
+
+	for _, item := range clusters.Items {
+		if item.Name == cluster {
+			log.Infof("K8s cluster %s exist and will not register it", cluster)
```

Comment:
operator already has a _ton_ of logging. I think logs like this only add clutter, not value.

Reply:
I think we should have a debug mode for the operator and only add info logs when running in debug mode. I find these logs helpful sometimes.